JSON/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "type": "object",
+    "properties": {
+        "tags": {
+            "type": "object",
+            "additionalProperties": true
+        },
+        "endpoint": {
+            "type": "object"
+        },
+        "tool-params": {
+            "type": "array"
+        },
+        "mv-params": {
+            "type": "object"
+        },
+        "passthru-args": {
+            "type": "string"
+        }
+    },
+    "required": [
+        "mv-params"
+    ]
```

Comment:
I think we probably want an `"additionalProperties": false` here, right?

Reply:
Agreed, the fact that the tool only allows specific values for --config <config> via code may not be enough